build.gradle.kts

```diff
@@ -119,6 +119,7 @@ dependencies {
 
     //General Utility
     implementation("net.sf.trove4j:trove4j:3.0.3")
+    // Match the dependency version of lavaplayer
     implementation("com.fasterxml.jackson.core:jackson-databind:2.10.1")
```

Comment:
We should update this to get rid of the warning on mvnrepository. Also, can you add a dependency on `jackson-core` as well?

Reply:
To maintain [compatibility with Lavaplayer](https://github.com/DV8FromTheWorld/JDA/pull/1819#discussion_r708700892), I updated the version to the same minor version, but to a version without CVE.
Matching jackson-core version added

Reply:
We [agreed](https://discord.com/channels/125227483518861312/869965829024915466/965660685700841533) to kill this CVE and upgrade to 2.13.2(.2). Lavaplayer will follow soon